hearth/lib/hearth/validator.rb

```diff
@@ -13,7 +13,7 @@ module Validator
     # @param context [String] The context of the value being validated.
     # @raise [ArgumentError] Raises when the value is not one of given type(s).
     def self.validate_types!(value, *types, context:)
-      return if !value || types.any? { |type| value.is_a?(type) }
+      return if value.nil? || types.any? { |type| value.is_a?(type) }
```

Comment:
Is this because of Boolean? We should make sure this is tested.

Reply:
Yup, won't raise when the input value is boolean `false` with a `String` as type

I'll add a test to cover this! 👍